zipkin-ui/js/component_ui/serviceName.js

```diff
@@ -18,7 +18,12 @@ export default component(function serviceName() {
   this.updateServiceNameDropdown = function(ev, data) {
     $('#serviceName').empty();
     this.$node.append($($.parseHTML('<option value="all">all</option>')));
-
+    // Sorting based on the localCompare so that sorting can be accomplished for non-ascii (non english) service names.
+    if (data.names) {
```

Comment:
now... is there a place for a unit test? :D

Reply:
Added them :)